### ISSUE TITLE
Options to disable syncing cursor status

### ISSUE THREAD

**Issue by coolwanglu** [mozilla/togetherjs#987]
While cursor syncing might be one of the essential features, I wonder if there could be an option for disabling it. My case is that mouse is not used at all, so all the syncing routine is a waste of resource of both clients and servers.

Also since mouse is not used, it looks weird with all the fingers/pointers showing on the page.

**Comment by nicholaswmin** [comment]
+1

**Comment by nicholaswmin** [comment]
+1

**Comment by mits2013** [comment]
I agree there should be way to disable them. In my case it is disturbing other flows.

**Comment by nicholaswmin** [comment]
@mits2013 The library is no longer maintained. I've switched to a Socket.io/Node.js setup myself.
